manager/dispatcher/assignments.go

```diff
@@ -45,6 +48,19 @@ func assignSecret(a *assignmentSet, readTx store.ReadTx, mapKey typeAndID, t *ap
 		}).Debug("failed to fetch secret")
 		return
 	}
+	// If a driver is used, give a unique ID per task to allow different values for different tasks
+	if doNotReuse {
```

Comment:
How does the statement "If a driver is used" map to the flag 'doNotReuse' ?

Reply:
Good catch, that comment was left from a previous implementation where secrets would never be reused if a secret driver was used.

Reply:
I'll change it to:
```golang
	// If the secret should not be reused for other tasks, give it a unique ID for the task to allow different values for different tasks.
```